.babelrc

```diff
@@ -1,11 +1,11 @@
 {
   "presets": [
-    "./tools/grommet-babel-preset-es2015",
```

Comment:
removing this is breaking the es6 distribution.

we run this

`cross-env BABEL_ENV=es6 babel ./src/js/ --ignore '__tests__' --out-dir ./dist/es6 --copy-files`

and `./tools/grommet-babel-preset-es2015` is the one responsible for adding `es6` as the babel environment.

You can test this by opening the `dist/es6/components/Accordion/Accordion.js`.

This is important for tree shaking